docs/CHANGELOG.md

```diff
@@ -19,6 +19,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Helper migrations function for injecting env vars into goose migrations. This was done to inject chainID into evm chain id not null in specs migrations.
 - OCR2 jobs now support querying the state contract for configurations if it has been deployed. This can help on chains such as BSC which "manage" state bloat by arbitrarily deleting logs older than a certain date. In this case, if logs are missing we will query the contract directly and retrieve the latest config from chain state. Chainlink will perform no extra RPC calls unless the job spec has this feature explicitly enabled. On chains that require this, nops may see an increase in RPC calls. This can be enabled for OCR2 jobs by specifying `ConfigContractAddress` in the relay config TOML.
 
+- Added multichain telemetry support. Each network/chainID pair must be configured using the new fields:
+  ```toml
+  [[TelemetryIngress.Endpoints]]
+  Network = '...' # e.g. EVM. Solana, Starknet, Cosmos
+  ChainID = '...' # e.g. 1, 5, devnet, mainnet-beta
+  URL = '...'
+  ServerPubKey = '...'
+  ```
+  These replace `TelemetryIngress.URL` and `TelemetryIngress.ServerPubKey`, which are no longer allowed.
```

Comment:
I'm worried that the runtime error won't be informative enough, if they overlook this note. I wonder if it would be possible to leave the old fields around so that they parse, but then fail more explicitly when they are actually set, with clear instructions to use the new fields instead :thinking: